proto/decentralized-vault/decentralized-vault.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+package ununifi.decentralizedvault;
+
+import "gogoproto/gogo.proto";
+import "decentralized-vault/params.proto";
+
+option go_package = "github.com/UnUniFi/chain/x/decentralized-vault/types";
+
+message TransferRequest {
+  string owner       = 1;
+  string eth_address = 2;
+  string nft_id      = 3;
+}
+
+message TransferRequestNft {
+  string id       = 1;
+  string uri      = 2;
+  string uri_hash = 3;
+  string owner    = 4 [
+    (gogoproto.moretags)   = "yaml:\"owner\"",
+    (gogoproto.customtype) = "github.com/UnUniFi/chain/types.StringAccAddress",
+    (gogoproto.nullable)   = false
+  ];
+  string ethAddress = 6;
```

Comment:
I think it would be nice to use snake case, `eth_address` to use same syntax as `uri_hash` to be consistent.